fv3_cap.F90

```diff
@@ -1040,10 +1052,13 @@ subroutine ModelAdvance_phase1(gcomp, rc)
     logical                     :: fcstpe
     character(len=*),parameter  :: subname='(fv3_cap:ModelAdvance_phase1)'
     character(240)              :: msgString
+    real(kind=8)                :: MPI_Wtime
 
 !-----------------------------------------------------------------------------
 
     rc = ESMF_SUCCESS
+    timep1rs = MPI_Wtime()
+    if(lprint) print *,'in fv3_cap,modeladvance phase1 time ', timep1rs-timep2re,mype
```

Comment:
Shouldn't this message be something like ' .. time between phase 2 and phase 1' like in ModelAdvance? We are subtracting here start of this subroutine from the end of the previous time step phase 2.

Reply:
Yes. changed.